lib/pyfrc/mains/cli_test.py

```diff
@@ -40,6 +41,10 @@ def run(self, options, robot_class, **static_options):
         from .. import config
         config.mode = 'test'
         config.coverage_mode = options.coverage_mode
+
+        # Enable all warnings - see PEP 565
+        if not sys.warnoptions:
+            warnings.simplefilter('default')
```

Comment:
The only thing I don't like about this is it'll show PendingDeprecatedWarnings that are in pytest as well as user code

Reply:
Will it? pytest is already imported by the time this executes.

The user code part is a reasonable objection though.

Reply:
I think it did for me when I had the old one. Because pytest itself called old methods? Until we see it happen we can ignore it I think

Reply:
I saw warnings within pytest from running `python -Wd`, but not with this. I don't expect this to be too much of an issue.